### PR TITLE
Fixes filenames for claim attachments

### PR DESCRIPTION
**Pull request by dan-palmer**
- [x] make all the changes

### CODE REVIEW

File: config/initializers/renderers.rb [hunk 1]
```diff
@@ -1,5 +0,0 @@
-ActionController::Renderers.add :pdf do |obj, options|
```
Comment (dan-palmer) [inline config/initializers/renderers.rb]: :door: :wave: :walking:

**Comment by dan-palmer** [comment]
Nah.. on second thoughts don't like this approach.

File: config/initializers/renderers.rb (hunk 1 above)
Comment (stevegraham) [inline config/initializers/renderers.rb]: See ya!

File: spec/models/claim_spec.rb [hunk 2]
```diff
@@ -239,39 +241,38 @@
...
-      expect(subject.attachments.size).to eq 1
+      expect(subject.attachments.size).to eq 2
     end
   end
 
   describe "#generate_pdf!" do
     context "pdf has already been generated" do
       it "does not attempt to generate another" do
-        allow(subject).to receive(:pdf_blank?).and_return false
+        subject = create :claim, :with_pdf
```
Comment (stevegraham) [inline spec/models/claim_spec.rb]: Vas?

File: app/services/carrierwave_filename.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+module CarrierwaveFilename
+  def for(attachment)
+    attachment.file.path.split('/').last rescue nil
```
Comment (stevegraham) [inline app/services/carrierwave_filename.rb]: Would it be better to use a nil guard rather than swallowing exceptions.

**Comment by dan-palmer** [comment]
@stevegraham removed that check on the pdf, didn't make much sense it being there & switched the nil munching out too.

**Pull request merged**